lang/en.json

```diff
@@ -571,4 +571,4 @@
     "defaultMessage": "Zoom out",
     "description": "Text label for zoom out button"
   }
-}
+}
```

Comment:
I think this is "sort.js" vs VS Code. Might be nice to fix but not worth worrying about now.